packages/core/test/assign.test.ts

```diff
@@ -367,56 +367,3 @@ describe('assign meta', () => {
     service.send({ type: 'EVENT' });
   });
 });
-
-it('should type events from onDone properly', () => {
```

Comment:
Is this test no longer valid?

Reply:
it's valid - i just moved this to `setup.types.test.ts`